identity/CHANGELOG.md

```diff
@@ -1,13 +1,18 @@
 ## 0.1.2 - unreleased
 
-- Follow Rust naming conventions for conversion methods.
-  See [PR 3775].
+-   Add `impl From<ed25519::PublicKey> for PublicKey` so that `PublicKey::from(ed25519::PublicKey)` works
+    See [PR 3805].
 
-[PR 3775]: https://github.com/libp2p/rust-libp2p/pull/3775
+[pr 3805]: https://github.com/libp2p/rust-libp2p/pull/3805
+
+-   Follow Rust naming conventions for conversion methods.
+    See [PR 3775].
+
+[pr 3775]: https://github.com/libp2p/rust-libp2p/pull/3775
 
 ## 0.1.1
 
-- Add `From` impl for specific keypairs.
-  See [PR 3626].
+-   Add `From` impl for specific keypairs.
+    See [PR 3626].
 
-[PR 3626]: https://github.com/libp2p/rust-libp2p/pull/3626
+[pr 3626]: https://github.com/libp2p/rust-libp2p/pull/3626
```

Comment:
Is all this reformatting necessary?

Reply:
No, it's not -- Prettier must be reformatting it without me noticing. I will manually undo it.